src/types/index.ts

```diff
@@ -1,49 +1,56 @@
-export interface LogOptions {
+export type LogOptions = {
```

Comment:
Go types!

Reply:
Why these changes?

Reply:
is this related in any sense to Dappnode?